node/cn/handler_test.go

```diff
@@ -192,64 +200,185 @@ func TestBroadcastBlockHash(t *testing.T) {
 	}
 }
 
-func TestBroadcastCNTx_CN_NotExists(t *testing.T) {
+func TestBroadcastTxsFromCN_CN_NotExists(t *testing.T) {
 	pm := &ProtocolManager{}
-	pm.nodetype = node.ENDPOINTNODE
+	pm.nodetype = node.CONSENSUSNODE
 	mockCtrl := gomock.NewController(t)
 	defer mockCtrl.Finish()
 
 	peers := newPeerSet()
 	pm.peers = peers
+	cnPeer, pnPeer, enPeer := createAndRegisterPeers(mockCtrl, peers)
 
-	cnPeer := NewMockPeer(mockCtrl)
-	pnPeer := NewMockPeer(mockCtrl)
-	enPeer := NewMockPeer(mockCtrl)
+	// Using gomock.Eq(txs) for AsyncSendTransactions calls,
+	// since transactions are put into a new list inside broadcastCNTx.
+	cnPeer.EXPECT().KnowsTx(tx.Hash()).Return(true).Times(1)
+	cnPeer.EXPECT().AsyncSendTransactions(gomock.Eq(txs)).Times(0)
```

Comment:
Does `.Times(0)` mean that it will not be called?

Reply:
Right. Actually it is not really needed but I leave them for clear representation.

Reply:
Then, is it meaningful to check a return value of the `AsyncSendTransactions` here?

Reply:
`AsyncSendTransactions` does not return anything and it is not even called. What's the point of checking the return value of the `AsyncSendTransactions`?

Reply:
Right. I confused with matchers. How about the matcher, `gomock.Eq(txs)` with `Times(0)`?

Reply:
As discussed f2f, gomock.Eq(txs) means nothing since Times(0) is harder condition.